Package.swift

```diff
@@ -10,6 +10,5 @@ let package = Package(
     targets: [
         .target(name: "UInt128"),
         .testTarget(name: "UInt128Tests", dependencies: ["UInt128"]),
-    ],
-    swiftLanguageVersions: [.v5]
```

Comment:
From [the docs](https://developer.apple.com/documentation/packagedescription/package/swiftlanguageversions) I understand you need to insert *all* compatible swift versions, not just the minimum one. I believe we can set the minimum swift version on line 1.

Reply:
So do you think by leaving it out we are stating support for all versions? The last landed PR made some backwards incompatible changes, so do you think it would put our users at risk to not explicitly state our supported Swift versions?

Reply:
I read up a bit and I think we'll be fine. Hopefully if it does impact any users they will report it.